src/main/web/templates/handlebars/partials/t3/related-articles.handlebars

```diff
@@ -18,9 +18,7 @@
                             </a>
                         </h4>
                         <p class="flush">
-                            {{#if description.summary}}{{description.summary}}{{/if}}
-                            {{#if description._abstract}}{{description._abstract}}{{/if}}
-
+                            {{#if description._abstract}}{{description._abstract}}{{description.metaDescription}}{{/if}}
```

Comment:
All looks good, I'll approve it now. Only thing is that it looks like we're missing an `{{else}}` here 👍